Description: Change makefile to build with hardening flags.
  This patch changes the makefile with hardening flags. It fixes lintian
  lintian tags hardening-no-relro and hardening-no-fortify-functions.
Author: Martin Steigerwald <ms@teamix.de>

--- a/Makefile
+++ b/Makefile
@@ -9,8 +9,12 @@
 DIR1= /test/dir1
 DIR2= /test/dir2
 
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+DPKG_EXPORT_BUILDFLAGS = 1
+include /usr/share/dpkg/buildflags.mk
+
 COBJS= fs_mark.o lib_timing.o
-CFLAGS= -O2 -Wall -D_FILE_OFFSET_BITS=64
+CFLAGS+= -O2 -Wall -D_FILE_OFFSET_BITS=64
 
 BIN=$(DESTDIR)/usr/bin
 
